src/Microsoft.Diagnostics.Runtime/src/Utilities/PEImage/FileVersionInfo.cs

```diff
@@ -2,47 +2,50 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
+
 namespace Microsoft.Diagnostics.Runtime.Utilities
 {
     /// <summary>
     /// FileVersionInfo reprents the extended version formation that is optionally placed in the PE file resource area.
     /// </summary>
     public sealed unsafe class FileVersionInfo
     {
-        // TODO incomplete, but this is all I need.  
+        /// <summary>
+        /// Gets the position of the string data within the resource block.
+        /// See http://msdn.microsoft.com/en-us/library/ms647001(v=VS.85).aspx
+        /// </summary>
+        public const int DataOffset = 0x5c;
+
         /// <summary>
         /// The verison string
         /// </summary>
-        public string FileVersion { get; }
+        public string? FileVersion { get; }
 
         /// <summary>
         /// Comments to supplement the file version
         /// </summary>
-        public string Comments { get; }
+        public string? Comments { get; }
 
-        internal FileVersionInfo(byte* data, int dataLen)
+        internal FileVersionInfo(Span<byte> data)
```

Comment:
Since there's no Span support without System.Memory, you could just implement this with a byte* like the old version.

Reply:
Done!